lib/ex_doc/retriever.ex

```diff
@@ -60,10 +91,15 @@ defmodule ExDoc.Retriever do
       raise Error, "module #{inspect(module)} is not defined/available"
     end
 
-    if docs_chunk = docs_chunk(module) do
-      generate_node(module, docs_chunk, config)
-    else
-      []
+    case docs_chunk(module) do
+      false ->
+        []
+
+      :hidden ->
+        %{module_doc_hidden: module}
```

Comment:
This also does not seem to be necessary. Seems to be left-over?

Reply:
fixed